CHANGES.txt

```diff
@@ -7,6 +7,8 @@ Next version (unreleased)
 - Add "highlight" in the query string of the URLs of search results.
   This parameter is understood by Sphinx-generated HTML files.
 
+- Add upload doc end point (to use dokang as a private python hosted).
```

Comment:
"documentation uploading end point" (to use **D**okang web frontend to serve the documentation).